apicast/src/get_token.lua

```diff
@@ -10,7 +12,7 @@ local function extract_params()
   params.authorization = {}
 
   if header_params['Authorization'] then
-    params.authorization = ngx.decode_base64(header_params['Authorization']:split(" ")[2]):split(":")
+    params.authorization = split(ngx.decode_base64(split(header_params['Authorization']," ")[2]),":")
```

Comment:
Is this a functional change, or just syntax?

We're trying to understand the changes that fix the broken OAuth....